tests/job_pytest.py

```diff
@@ -1,12 +1,41 @@
 from src.job import Job
-from src.load_profile import LoadProfile
+import pandas as pd
+import pytest
 
 
 def test_to_message():
-    load_profile = LoadProfile([0.0, 3600.0], [0.0, 1.0])
+    load_profile = pd.Series(index=[0.0, 3600.0], data=[0.0, 1.0])
     job = Job(0, 1, load_profile)
 
     expected = dict(est=0, lst=1, load_profile=load_profile)
     actual = job.to_message()
 
     assert expected == actual
+
+def test_power():
+    job1 = Job(1, 2, pd.Series(index=[0.0, 1.0, 3.0, 4.0], data=[0.0, 10.0, 30.0, 40.0]))
+    job2 = Job(0, 1, pd.Series(index=[0.0, 3600.0], data=[0.0, 1.0]))
+
+    expected1 = 0.0
+    expected2 = 0.0
+    expected3 = pytest.approx(15.0, 0.1)
+    expected4 = pytest.approx(23.0, 0.1)
+    expected5 = 40.0
+    expected6 = 40.0
+    expected7 = pytest.approx(0.5, 0.1)
+
+    actual1 = job1.power(-1)
+    actual2 = job1.power(0)
+    actual3 = job1.power(1.5)

```

Comment:
The job has no value for `t=1.5`, so it needs to interpolate. Since `power(t=1)=10` and `power(t=3)=30`, the interpolated value for `power(1.5)` is 15.